src/main/asciidoc/reference/mongo-repositories.adoc

```diff
@@ -281,6 +281,10 @@ lower / upper bounds (`$gt` / `$gte` & `$lt` / `$lte`) according to `Range`
 | `Exists`
 | `findByLocationExists(boolean exists)`
 | `{"location" : {"$exists" : exists }}`
+
+| `IgnoreCase`
+| `findByUsernameIgnoreCase(String username)`
+| `{"username" : {"$strcasecmp" : username }}`
```

Comment:
`$strcasecmp` is an aggregation operator. The `MongoQueryCreator` uses a `Pattern` that translates to `{ $regex : '^username$', '$options' : 'i' }`.

Reply:
Thank you for the feedback. I will make the changes.

Reply:
Hi @christophstrobl. I have made the changes. You were right, that regex is used in the code.
Is it ok to use double quotes instead of single quotes? I noticed that strings are used and even the i from the options is a String.